app/src/marketplace/components/deprecated/Hero/hero.pcss

```diff
@@ -0,0 +1,42 @@
+.hero {
+  background: var(--greyLight3);
+  padding: 2.5em 0;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 1.75rem;
+}
+
+@media (--md-up) {
+  .container {
+    grid-template-columns: 380px 1fr;
+    grid-row-gap: 0;
+    grid-column-gap: 3rem;
+  }
+}
+
+@media (--md-down) {
+  .heroRow {
```

Comment:
Grepped for `heroRow` and the only thing I found is styling. I know it's marked _deprecated_ but maybe we can remove it already?